src/pybind/mgr/dashboard/settings.py

```diff
@@ -20,6 +20,9 @@ class Options(object):
     ENABLE_BROWSABLE_API = (True, bool)
     REST_REQUESTS_TIMEOUT = (45, int)
 
+    # AUTHENTICATION ATTEMPTS
+    ACCOUNT_LOCKOUT_ATTEMPTS = (10, int)
```

Comment:
It would be better we update the document about how to get or set the attempt and how to re-enable a locked user.
```
ceph dashboard get-account-lockout-attempts
ceph dashboard set-account-lockout-attempts <value:int> :  Set the ACCOUNT_LOCKOUT_ATTEMPTS option value
dashboard ac-user-enable <username> :  Enable a user
```

Reply:
Yup. Will do it. Just to clarify, it can be added after this right https://docs.ceph.com/en/latest/mgr/dashboard/#username-and-password
@bk201

Reply:
This will be a nice place to add. Thanks!

Reply:
I am yet to update this. I'll update it ASAP. @bk201

Reply:
Updated @bk201

Reply:
Thanks!